uncertainty_toolbox/utils.py

```diff
@@ -2,14 +2,22 @@
 Util functions for the toolbox.
 """
 
+import sys
 import numpy as np
 
 
 def assert_is_flat_same_shape(*args):
-    assert isinstance(args[0], np.ndarray)
-    first_shape = args[0].shape
-    for arr in args:
-        assert isinstance(arr, np.ndarray)
-        assert len(arr.shape) == 1
-        assert arr.shape == first_shape
+    """Check if inputs are all same-length 1d numpy.ndarray."""
+
+    try:
+        assert isinstance(args[0], np.ndarray), "Error: all inputs must be of type numpy.ndarray"
+        first_shape = args[0].shape
+        for arr in args:
+            assert isinstance(arr, np.ndarray), "Error: all inputs must be of type numpy.ndarray"
+            assert len(arr.shape) == 1, "Error: all inputs must be 1d numpy.ndarray"
+            assert arr.shape == first_shape, "Error: all inputs must have the same length"
+    except AssertionError as error_msg:
+        print(error_msg)
```

Comment:
I think doing this will just print the message and not give a traceback? If we want to more detailed about the error, I would argue that we switch from asserts to raising errors. I would think that all these would be TypeErrors? Although I am not sure if there is a more suitable one.